content/en/docs/installation.md

```diff
@@ -476,7 +476,7 @@ resources:
 - gotk-components.yaml
 - gotk-sync.yaml
 patches:
-  - path: psp-patch.yaml
+  - path: ../psp-patch.yaml
```

Comment:
The psp-patch file should in the same dir (`flux-system`) as this kustomization. Please update the path above to `clusters/my-cluster/flux-system/psp-patch.yaml` and undo this change.